.gitignore

```diff
@@ -27,6 +27,7 @@
 .scalastyle
 .settings
 .cache*
+.tmpBin
```

Comment:
Is this a temporal bin in which I can collect all those short time snippets that are not properly used to recycle them at the end of the day? That would be nice …